### ISSUE TITLE
Predeclared should be ./... not .

### ISSUE THREAD

**Issue by bufdev** [jhump/protoreflect#323]
```
$ make
go get -d -v -t ./...
gofmt -s -l .
go vet ./... --ignore internal/testprotos
predeclared .
-: no Go files in /home/foo/dev/protoreflect
predeclared: error during loading
make: *** [predeclared] Error 1
```

Makefile should have `predeclared ./...` not `predeclared .` (I could change it, but figured you'd might be able to quickly!)

**Comment by jhump** [comment]
LOL, I was going through all of my repos fixing this. I just merged a fix for this #322

**Comment by bufdev** [comment]
lol
